assets/static/img/logo-reana.svg

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
The PNG version of the logo has nicer "top triangle workflow split" than the SVG version; can you please amend?  

It would be also good to create a favicon out of it...

Reply:
how about this version with a bit of curve:
![reana](https://user-images.githubusercontent.com/717470/39998593-d51809c8-5786-11e8-99a0-4a9d21db03b4.png)

Reply:
Personally I like straight lines more, as they seem to better remind of typical workflow graphs.... Perhaps we could create a separate RFC issue with several logo ideas (workflows, recycle arrows, etc?)